solr/contrib/ltr/src/java/org/apache/solr/ltr/rest/ManagedModelStore.java

```diff
@@ -189,10 +189,6 @@ public LTRScoringAlgorithm makeLTRScoringAlgorithm(Map<String,Object> map)
             (Map<String,Object>) modelFeature;
         final Feature feature = parseFeature(modelFeatureMap,
             fstore);
-        if (!fstore.containsFeature(feature.getName())) {
-          throw new ModelException("missing feature " + feature.getName()
```

Comment:
We should also add a unit test that verifies the above since I guess it did not catch it for you.